### PR TITLE
footer added

### PR DESCRIPTION
**Pull request by Hiresh23**
footer added

### CODE REVIEW

**Comment by Hiresh23** [comment]
Screenshot 2022-10-12 111936

**Comment by Dhruv031003** [comment]
@Hiresh23 
- The colors are bit off, check the figma file for colors and apply the changes accordingly.
- The Donaate logo is also not aligned properly

**Comment by Hiresh23** [comment]
Screenshot 2022-10-12 131310
is this ok

**Comment by Dhruv031003** [comment]
@Hiresh2
- Donaate is a logo you can find it in the Donaate-UI folder.
- About, Donation and FAQ's are too bold apply normal style to them

**Comment by Dhruv031003** [comment]
@Hiresh23 any update??...are you still on this or not.

**Comment by Hiresh23** [comment]
yes yess ok i will do it now

**Comment by Dhruv031003** [comment]
@Hiresh23 Please resolve the conflicts.

**Comment by Hiresh23** [comment]
done

File: index.html [hunk 1]
```diff
@@ -5,6 +5,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Donaate</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
```
Comment (Dhruv031003) [inline index.html:13]: There are two body tags and navbar fix this

**Review by Dhruv031003 — CHANGES_REQUESTED** [review]

**Comment by Dhruv031003** [comment]
> @Hiresh23 Please resolve the conflicts.
.

File: index.html (hunk 1 above)
Comment (Hiresh23) [reply index.html:13]: meaning

Comment (Dhruv031003) [reply index.html:13]: If you see the index.html file there is a body opening tag on line 13. But then there is a head closing tag on line 35 and then again a body opening tag.
- The code in between these lines is also repeated two times.

Comment (Hiresh23) [reply index.html:14]: Done you can check now

**Comment by mr-palindrome** [comment]
Hey @Hiresh23 !
While fixing the conflicts you have removed the existing codes that were added in the last PR. Please add back those code and update your PR.

**Comment by Hiresh23** [comment]
In which file html or css

**Comment by Dhruv031003** [comment]
@Hiresh23 
After you forked we had new Hero section made. You removed the new Hero section(The section with image of a man) rather than removing the old one while resolving the merge conflicts.
To fix this you can fork this again in a seperate place and then add Footer in that and push it again.

**Comment by Hiresh23** [comment]
ok

**Comment by Hiresh23** [comment]
new pull request made should i close this one

**Pull request closed without merging**